data/transactions/logic/README.md

```diff
@@ -275,6 +275,8 @@ Some of these have immediate data in the byte or bytes after the opcode.
 | 53 | GlobalNumByteSlice | uint64 | Number of global state byteslices in ApplicationCall. LogicSigVersion >= 3. |
 | 54 | LocalNumUint | uint64 | Number of local state integers in ApplicationCall. LogicSigVersion >= 3. |
 | 55 | LocalNumByteSlice | uint64 | Number of local state byteslices in ApplicationCall. LogicSigVersion >= 3. |
+| 56 | AppProgramExtraPages | uint64 |  |
+| 57 | CreatableID | uint64 | The assigned creatable ID for an asset or application creation transaction, or 0 if the transaction does not create anything. LogicSigVersion >= 4. |
```

Comment:
Are you allowed to look forward to a later slot?  It's nearly possible to do so (since the id is determined by the sequence number of the txn), but I don't think it's a good idea (since I want to make it possible to create an asa in teal - soon it won't be possible to know if an createable will be made or not).  Anyway, if you can't, we should document it, as usually you can look forward with gtxn.

Reply:
I'll add errors to make sure you can't if you are going to make it impossible soon anyways. I was also wondering what should happen with LogicSigs? Will this txn field work from within stateless contracts?